### PR TITLE
[Gluten-5152][CH]Support Optimize and VACUUM command for clickhouse tables

### PR DESCRIPTION
**Pull request by binmahone**
## What changes were proposed in this pull request?

(Please fill in changes proposed in this fix)

(Fixes: #5152 )

## How was this patch tested?

(Please explain how this patch was tested. E.g. unit tests, integration tests, manual tests)

(If this patch involves UI changes, please attach a screenshot; otherwise, remove this)

### CODE REVIEW

File: cpp-ch/local-engine/Operator/ExpandTransform.cpp [hunk 1]
```diff
@@ -15,6 +15,8 @@
  * limitations under the License.
  */
 #include <memory>
+
+#include "Columns/ColumnSparse.h"
```
Comment (liuneng1994) [inline cpp-ch/local-engine/Operator/ExpandTransform.cpp:19]: use <>

**Review by liuneng1994 — COMMENTED** [review]
general LGTM

**Review by zzcclp — APPROVED** [review]
LGTM

File: cpp-ch/local-engine/Operator/ExpandTransform.cpp (hunk 1 above)
Comment (binmahone) [reply cpp-ch/local-engine/Operator/ExpandTransform.cpp:19]: will fix in my another PR. Let's avoid running another round of CI

**Pull request merged**